controllers/groups_controller.js

```diff
@@ -0,0 +1,101 @@
+const db = require('../db');
+const sql = require('../sql');
+
+exports.index = (req, res) => {
+  db.query(sql.groups.queries.get_groups, (err, data) => {
+    if (err) console.error('Cannot get groups');
+    res.render('groups', { data: data.rows });
+  });
+};
+
+exports.new = (req, res) => {
+  db.query(sql.courses.queries.get_courses, (err, courses) => {
+    if (err) console.error('Cannot get courses');
+    res.render('groupNew', { courses: courses.rows });
+  });
+};
+
+exports.create = (req, res) => {
+  const { name } = req.body;
+  const { semester_name } = req.body;
+  const { module_code } = req.body;
```

Comment:
Can be simplified the three lines to this I think?
```suggestion
  const { name, semester_name, module_code } = req.body;
```